ls1intum/Artemis
# [Feature] Instructor rating dashboard

### Checklist

### Motivation and Context


Students have the opportunity to rate their assessments. This PR enables Instructors to see the ratings within their course. Aggregation and grouping of Ratings will be done in a follow-up.

### Description

- Added Rating List
- Added link to Rating List in Instructor Course Dashboard

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Management
3. Navigate to Instructor Dashboard
4. See ratings in "Information about the course"
5. Click on "Ratings"
6. See rating list
7. Check that sorting works
8. Check that "Show the submission" button works

### Test Coverage

* rating-list.component.ts: 94.74%
* rating.service.ts: 100%
* Rating Resource: 100%
* RatingService.java: 100%
* text-editor.component.ts: 84.17%

### Screenshots



Instructor Dashboard:
![image](https://user-images.githubusercontent.com/51994246/91427269-f1874c80-e85d-11ea-8dc3-ce3940825a02.png)

Rating List:
![image](https://user-images.githubusercontent.com/51994246/94060719-b0df0c80-fde4-11ea-890e-e19ab3ea292e.png)
